.Q.prf0 won't stop the process with negative pid (#149)
KxSystems/docs: docs/ref/dotq.md

This process must be started from the same binary as the one running `.Q.prf0`, otherwise `binary mismatch` is signalled.

> Since 4.1t 2022.03.25, .Q.prf0 won't try to stop the process if passed a negative pid. Useful for calls in the middle of debugging, etc.

Reviewer: ```suggestion
Since 4.1t 2022.03.25, .Q.prf0 will not try to stop the process if passed a negative `pid`. Useful for calls in the middle of debugging, etc.
```
